IDE/MonoDevelop/MonoDevelop.MonoGame/templates/Common/Program.cs

```diff
@@ -13,69 +13,69 @@
 
 namespace ${Namespace}
 {
-	#if __IOS__ || __TVOS__
-	[Register("AppDelegate")]
-	class Program : UIApplicationDelegate
-	#else
-	static class Program
-	#endif
+    #if __IOS__ || __TVOS__
+    [Register("AppDelegate")]
+    class Program : UIApplicationDelegate
+    #else
+    static class Program
+    #endif
     {
-        private static Game1 game;
-
-		internal static void RunGame() 
-		{
-			game = new Game1();
-			game.Run();
-		}
-			
+        internal static void RunGame() 
+        {
+            using (var game = new Game1())
+            {
+                game.Run();
```

Comment:
@cra0zy unfortunately this change breaks iOS :( Because on iOS we don't stay in the Run method until the game exits we exit immediately. As a result game is free'd and boom!

We'll need to go back to static.